public_html/wp-content/themes/pattern-directory/src/components/pattern-grid/index.js

```diff
@@ -21,10 +22,8 @@ function PatternGrid() {
 
 	return (
 		<>
-			<header
-				style={ { background: 'whitesmoke', padding: '24px', textAlign: 'center', margin: '0 0 24px' } }
-			>
-				Section filters
+			<header>
+				<PatternGridMenu />
```

Comment:
Looking at this in action, I think this should be a step higher (out of the `header`), and the context label (which is a header) should be the part in a header tag, while the menu should be in a `nav` - like this:

<img width="1035" alt="Screen Shot 2021-04-07 at 6 19 26 PM" src="https://user-images.githubusercontent.com/541093/113941793-ca1e3800-97cd-11eb-8802-9db0003cdd14.png">

Reply:
Yeah, good call. I got caught up in the weeds. That makes a lot more sense.